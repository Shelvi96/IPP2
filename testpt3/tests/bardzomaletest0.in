NEW ID

757:0912>27;:0644
83>2227775666
3147:999;8>28;65:78
9224199>385:9
5036>232832
002803;2>008074:66;
:2:;95>904
1821954>49319;9;97
45965:;;>43853
604;490;21>31;;488
21>2
0:;83>;1
44;54;;612>513128
@09847752244:009950836;
@;45465;2:926813348;20
@3:84:;:9116:508466584628968008
@3:8;9;9:124375;43910272280:806
@79076;8
@439826;691017;798778;50;20:04
@58168408219274225220259
@40724;5986
@4:;411578112634::281600140
@6001831186676898605
@:71355469847899402908872842297
@;20385;67;;278
@3051051;794:;59;
