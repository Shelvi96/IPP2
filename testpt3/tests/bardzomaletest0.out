2377756965
235648347
555646475696616777
2419126033589600256
0
201593836132466688
3529831122
0
37663732329036
0
509696460521374005
8
90
